src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,100 @@
+
+Before you start, talk to the Nerd Bytes team and see if your video content relates to something they've already worked on, or plan on working on.
+
+## Some tips on production value [#production-value]
+
+Some thoughts on production value and how long we take on these projects. 
+
+Be okay with some imperfection. We are not video production professionals and we don't believe anyone expects us to be. A very quickly and roughly made video can still show how to use the UI and still benefit people even without high production value.
+
+Another reason we are okay with some lower production quality: we know that most of our videos will go out-of-date sometime in future, sometimes in very near future. For this reason, we see it as better to err on side of getting helpful videos out quickly as opposed to spending a lot of time on videos that we know we'll have to replace.  
+
+Don't let your quest for perfection be at odds with getting out a highly useful video on quicker side. 
+
+To take some common examples of imperfection that can bother producers but that aren't that big of a deal: 
+
+* A quickly written and produced video can be completely fine, provided you think it will add obvious value.  
+* Some rough spots in your voice-over (uhs, umms, pauses) are okay.
+* Some background audio hiss or noise can be okay, if not too bad. 
+* Some rough/imperfect cuts between clips (for example, a cut where a cursor suddenly jumps somewhere else) can be okay. 
+
+## Scripting tips [#script-tips]
+
+Consider talking first to SMEs who know about that UI. Consider talking to UX designers who may know about best practices for a specific UI or about upcoming changes that might make your video out-of-date in near future. 
+
+Try to keep your video under two minutes. If your script is longer than that, consider breaking it up into more than one script. 
+
+You can estimate your video length by doing a leisurely read-through of your script.
+
+You don't have to read your script verbatim. It is completely fine to do a relaxed, casual, non-scripted narration, if you feel comfortable doing that. Or a loose read-through of your script, where you read some of it but then digress as feels natural. 
+
+## Tips for recording [#recording-tips]
+
+Tips for using Camtasia to record your screen capture and voice-over audio: 
+
+* Record in a quiet room, using some sort of external microphone (for example, Apple headphones). Don't use the built-in computer microphone. 
+* To make it easier to read your script, you can place a second computer display with your script on it directly beside the computer display you'll be recording on.
+* To record: in Camtasia, go to **File > New recording**. You can use the **Full screen** option to record your entire screen (and if you do that, you can crop your video when editing). Also ensure your audio input is set correctly. To test that everything is working, record a minute or so and then review the video. 
```

Comment:
@zuluecho9 it sounds like you can either narrate while you're recording the cursor movements, but do some people just record the movements first, and then record the audio separately and overlay it?